aea/cli/upgrade.py

```diff
@@ -118,6 +133,7 @@ def upgrade_project(ctx: Context) -> None:  # pylint: disable=unused-argument
 
     if not items_to_upgrade:
         click.echo("Everything is already up to date!")
+        click.echo("\n")
```

Comment:
Why not:
```suggestion
        click.echo("Everything is already up to date!\n")
```

Reply:
that was an unwanted change; will remove